tac/agents/v1/agent.py

```diff
@@ -120,6 +126,14 @@ def stop(self) -> None:
         self.liveness._is_stopped = True
         self.mail_box.stop()
 
+    @abstractmethod
+    def setup(self) -> None:
+        """
+        Set up the agent.
+
+        :return: None
+        """
```

Comment:
Can we add `pass`?

Reply:
Maybe `self.mail_box.start()` should be called here rather than in start. As this then decouples the starting of the mailbox from the starting of the agent (e.g. I can have an agent with a different setup method).